src/snowflake/cli/_app/snow_connector.py

```diff
@@ -71,24 +89,20 @@ def connect_to_snowflake(
 
     if connection_name:
         connection_parameters = get_connection_dict(connection_name)
-        connection_parameters = get_connection_dict(connection_name)
```

Comment:
is there a side-effect to calling this twice, or was this just a bug?

Reply:
No side effects, I presume it was a bad merge.